monai/transforms/lazy/utils.py

```diff
@@ -223,7 +223,9 @@ def resample(data: torch.Tensor, matrix: NdarrayOrTensor, kwargs: dict | None =
         img.affine = call_kwargs["dst_affine"]
         return img
 
+    # TODO: lazy evaluation - no need to separately set lazy to False
+    # resampler = monai.transforms.SpatialResample(lazy=False, **init_kwargs)
```

Comment:
Please remove commented code and the `TODO` mark.

Thanks.

Reply:
Fixed in next commit